### ISSUE TITLE
Comment color coding breaks when comment follows String and contains quote(s) and another oddity

### ISSUE THREAD

**Issue by IanBellomy** [koenbok/Framer#464]
<img width="252" alt="screen shot 2017-01-27 at 9 33 19 pm" src="https://cloud.githubusercontent.com/assets/17239616/22393624/4ddb4bfe-e4d8-11e6-865e-d88a66b6efff.png">

**Comment by nvh** [comment]
Heh, that's a bug :)
/cc @onnlucky

**Comment by onnlucky** [comment]
Yeah, but it is a known issue, it shows the limits of our current highlighting strategy. If it helps, it will not break if one side uses single quotes, and the other side double quotes, e.g. this does show up correctly: `"123" # '123'`

**Comment by IanBellomy** [comment]
If it matters, this came up when I was making instructional materials and needed to distinguish between variables, key words, and strings. Simple example:

    word = "cat"    #  'word' is a variable that contains the string "cat".
